config/initializers/filter_parameter_logging.rb

```diff
@@ -41,19 +43,24 @@
   page
   persistent_attachment_id
   qqtotalfilesize
+  queue_time
   reply_id
+  result
   saved_claim_id
   service
   showCompleted
   size
   sort
   startDate
   startedFormVersion
+  statsd
   status
   submission_id
   tags
   tempfile
+  time_to_transition
```

Comment:
I am curious how this one is still getting through, if this is only being added here now 🤔

Reply:
@kylesoskin Hmmm maybe for some widgets its not `@payload.time_to_transition` and others there are?